### ISSUE TITLE
[Help] Test item history API does not return history of setup methods

### ISSUE THREAD

**Issue by mchundi** [reportportal/reportportal#1946]
My regression suite has methods with "BeforeSuite" annotation. When I try to retrieve the history of tests using test name of setup methods, it does not return any data.
API used:
api/v1/polaris/item/history?filter.eq.launchId=<>&filter.eq.name=setup&historyDepth=30&isLatest=true&page.size=100
UI does show history.
How can I get the history of BeforeSuite methods?

**Comment by alena-babrova** [comment]
Hi @mchundi,

I can suggest you reuse the requests that are utilized by UI. You can find everything in the browser's dev tools.
For example, below is the way you can get history of "Before Method":
_api/v1/<project>/item/history?historyDepth=10&type=table&**filter.in.type=BEFORE_METHOD**&page.page=1&page.size=20&filter.eq.parentId=3098113048_

![image](https://github.com/reportportal/reportportal/assets/89584140/8238012e-9329-4110-96b2-30665cd680ec)

Basically, this approach can be applicable to anything which is done in UI.
Our UI relies on the same API, so you can always search for examples in browser.

I hope this information will be helpful for you.
